lib/vrl/stdlib/src/random_float.rs

```diff
@@ -0,0 +1,133 @@
+use ::value::Value;
+use rand::{thread_rng, Rng};
+use std::ops::Range;
+use vrl::prelude::*;
+
+const INVALID_RANGE_ERR: &str = "max must be greater than min";
+
+fn random_float(min: Value, max: Value) -> Resolved {
+    let min = min.try_float()?;
+    let max = max.try_float()?;
+
+    if max <= min {
+        return Err("max must be greater than min".into());
```

Comment:
Should this be using `INVALID_RANGE_ERR` ? :)
In fact, the `max <= min` check is done twice, one here and another in `get_range()`.
Maybe the check here is actually not necessary because the error would propagate?

The implementation in `random_int.rs` looks cleaner than this one.